Readme.md

```diff
@@ -5,10 +5,21 @@ Compatible with Node.js and web browsers.
 
 ## How to use
 
+First, you'll need a WordPress.com REST API OAuth2 token:
+- [OAuth2 Authentication](https://developer.wordpress.com/docs/oauth2/)
+  at WordPress.com Developer Resources
+- [`wpcom-oauth-cors`](https://github.com/Automattic/wpcom-oauth-cors):
+  a client-side WordPress.com OAuth2 library using CORS
+- [`wpcom-oauth`](https://github.com/Automattic/node-wpcom-oauth):
+  a server-side (Node.js) WordPress.com OAuth2 library
+- If you just want to create a token, the
+  [example app bundled with `wpcom-oauth`](https://github.com/Automattic/node-wpcom-oauth/tree/master/example)
+  is probably the quickest way.
+
 ### Node.js
 
-Introduce the `wpcom` dependency in your `package.json`, and
-then initialize it with your API Token.
+Introduce the `wpcom` dependency into your `package.json`, and then initialize
+it with your API token.
```

Comment:
If we're going to say that the API token is optional, shouldn't we provide some guidance about which situations require a token and which ones do not?